.github/workflows/npm-publish.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       uses: actions/checkout@v3
     - uses: actions/setup-node@v3
       with:
-        node-version: 14
+        node-version: 20
```

Comment:
Can you use the `strategy` here as well?

Like in `.github/workflows/github-publish.yml` file.